packages/component-library/draft/Kaizen/TitleBlock/TitleBlock.scss

```diff
@@ -46,7 +46,7 @@
   @include ca-type-ideal-small;
   display: flex;
   position: absolute;
-  z-index: $ca-z-index-popover;
+  z-index: $ca-z-index-sticky;
```

Comment:
The reason I chose `$ca-z-index-sticky` is that it's `1020`, to make it lower than `$ca-z-index-fixed`, which is `1030` and is what the navigation bar uses.

Reply:
I might be missing context here, but why does the title block need an elevated z-index?